fenixedu-ist-integration/src/main/java/pt/ist/fenixedu/integration/ui/FenixEduISTLegacyContextListener.java

```diff
@@ -227,6 +230,25 @@ public String uVATNumberFor(final Person person) {
                 return warnings;
             };
         });
+
+        Signal.registerWithoutTransaction("academic.candidacy.registration.created",
+                new Consumer<RegistrationCreatedByCandidacy>() {
+                    @Override
+                    public void accept(final RegistrationCreatedByCandidacy candidacy) {
+                        createEvents(candidacy);
+                    }
+
+                    @Atomic
```

Comment:
This is always write right? If so add `mode = TxMode.WRITE` (avoids the first restart)